### PR TITLE
NetworkManager: Make sure we show an IM at the *start* of a Wi-Fi toggle action

### PR DESCRIPTION
**Pull request by NiLuJe**
There's possibly a weird behavioral quirk on Kobo depending on whether there are system-level saved networks or not, c.f., #8667,
so this ensures that the worst-case scenario gets a hint that something is indeed actually happening ;).

Heavily based on @zwim's implementation in #8667, just took a more defensive approach to refresh races to avoid glitches on mxcfb ;).

(First commit is a random comment cleanup I missed in #9036 ;)).

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/9064)

### CODE REVIEW

**Review by Frenzie — APPROVED** [review]
Alright

**Comment by zwim** [comment]
:+1

**Comment by yparitcher** [comment]
While we are dealing with WiFi, a slight hijack, on my kindle when the WiFi is on but not connected the `ToggleWiFi` gesture tries to connect so i have no way of turning it off. For eample I use the gesture to turn it on, then leave the range I can't use it to turn off. I think the more direct approach would be to toggle On -> Off states so if i am disconnected and want to reconnect i can always go Off then On, but I am able to turn it off.

TLDR: the `tooglewifi` gesture should check for `iswififup` not `isconnected`

**Comment by NiLuJe** [comment]
(Rebased)
